system/jlib/jflz.cpp

```diff
@@ -676,6 +682,9 @@ class jlib_decl CFastLZCompressor : public CInterface, public ICompressor
 
     virtual void open(void *buf,size32_t max)
     {
+        wrmax = max / 4;
+        if (wrmax < 256)
+            wrmax = max;
```

Comment:
just seemed odd that:
if max is 1024, wrmax = 256
if max is 1023, wrmax = 1023